docker/Makefile

```
@@ -38,9 +38,9 @@ samtools.$(TAG): Dockerfile
 	--build-arg DOCKER_TAG=$(TAG) \
 	--build-arg IRODS_VERSION=4.2.11 \
 	--build-arg BATON_VERSION=4.2.1 \
-	--build-arg HTSLIB_VERSION=1.19.1 \
-	--build-arg SAMTOOLS_VERSION=1.19.2 \
-	--build-arg BCFTOOLS_VERSION=1.19 \
+	--build-arg HTSLIB_VERSION=1.20 \
+	--build-arg SAMTOOLS_VERSION=1.20 \
+	--build-arg BCFTOOLS_VERSION=1.20 \
```

Should we add deflate here just for symmetry?
```suggestion
	--build-arg BCFTOOLS_VERSION=1.20 \
	--build-arg DEFLATE_VERSION=1.20 \
```

---

That, or just remove all of them so versions are only in one place...